From b0db0a30e6fe1e4ea2b089f2b94b985bc3c89139 Mon Sep 17 00:00:00 2001
From: Yu KANECHIKA <yu.kanechika.uw@renesas.com>
Date: Thu, 4 Mar 2021 04:05:53 +0000
Subject: [PATCH] mmngr_drv: mmngr: Update PA-VA mapping table for HiHope
 RZ/G2N

HiHope board with RZ/G2N have 4 GB SDRAM for LPDDR4-CH0.
This commit assign 3rd and 4th PA-VA mapping table entry
for not-shadowed 2 GB part of 4 GB.

  - 3rd entry: 0x4_8000_0000 to 0x4_BFFF_FFFF
  - 4th entry: 0x4_C000_0000 to 0x4_FFFF_FFFF

Signed-off-by: Yu KANECHIKA <yu.kanechika.uw@renesas.com>
---
 .../mmngr/mmngr-module/files/mmngr/include/mmngr_private.h    | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/mmngr_drv/mmngr/mmngr-module/files/mmngr/include/mmngr_private.h b/mmngr_drv/mmngr/mmngr-module/files/mmngr/include/mmngr_private.h
index cea4271..e263429 100644
--- a/mmngr_drv/mmngr/mmngr-module/files/mmngr/include/mmngr_private.h
+++ b/mmngr_drv/mmngr/mmngr-module/files/mmngr/include/mmngr_private.h
@@ -302,10 +302,10 @@ static int validate_memory_map(void);
 #define M3_IPMMU_ADDR_SECTION_2	0x0080000000ULL
 #define M3_IPMMU_ADDR_SECTION_3	0x0600000000ULL
 /* Table entries for M3N */
-#define M3N_IPMMU_ADDR_SECTION_0	0x0
+#define M3N_IPMMU_ADDR_SECTION_0	0x04C0000000ULL
 #define M3N_IPMMU_ADDR_SECTION_1	0x0040000000ULL
 #define M3N_IPMMU_ADDR_SECTION_2	0x0080000000ULL
-#define M3N_IPMMU_ADDR_SECTION_3	0x0
+#define M3N_IPMMU_ADDR_SECTION_3	0x0480000000ULL
 /* Table entries for E3 */
 #define E3_IPMMU_ADDR_SECTION_0	0x0
 #define E3_IPMMU_ADDR_SECTION_1	0x0040000000ULL
-- 
2.26.2

